source/_lovelace/markdown.markdown

```diff
@@ -36,3 +36,25 @@ content: >
 
   Starting with Home Assistant 0.72, we're experimenting with a new way of defining your interface. We're calling it the **Lovelace UI**.
 ```
+
+## Template variables
+
+A special template variable - `config` is set up for the `content` of the card. It contains the configuration of the card.
+
+E.g.
+```yaml
+type: entity-filter
+entities:
+  - light.bed_light
+  - light.ceiling_lights
+  - light.kitchen_lights
+card:
+  type: markdown
+  content: |
+    The lights that are on are:
+    {% for l in config.entities %}
+      - {{ l.entity }}
+    {%- endfor %}
+
+    And the door is {% if is_state('binary_sensor.door', 'on' %} open {% else %} closed {% endif %}.
```

Comment:
You need to put `{% raw %}` tags around your templates. 

Also, the rest of the docs don't mention at all that templates are allowed ??

Reply:
Uh... yes it does? What's going on here?
It's in the code on the next branch: https://github.com/home-assistant/home-assistant.io/blame/next/source/_lovelace/markdown.markdown#L21
but not shown in the diff?

Reply:
Fixed.
I have no idea why git got that wrong...